.eslintrc.js

```diff
@@ -827,98 +833,6 @@ module.exports = {
-          },
-          {
-            name: '@testing-library/react-hooks',
-            message:
-              'Please import from `sentry-test/reactTestingLibrary` instead so that we can ensure consistency throughout the codebase',
-          },
-          {
-            name: '@testing-library/user-event',
-            message:
-              'Please import from `sentry-test/reactTestingLibrary` instead so that we can ensure consistency throughout the codebase',
-          },
-          {
-            name: '@sentry/browser',
-            message:
-              'Please import from `@sentry/react` to ensure consistency throughout the codebase.',
-          },
-          {
-            name: 'marked',
-            message:
-              "Please import marked from 'app/utils/marked' so that we can ensure sanitation of marked output",
-          },
-          {
-            name: 'lodash',
-            message:
-              "Please import lodash utilities individually. e.g. `import isEqual from 'lodash/isEqual';`. See https://github.com/getsentry/frontend-handbook#lodash from for information",
-          },
-          {
-            name: 'lodash/get',
-            message:
-              'Optional chaining `?.` and nullish coalescing operators `??` are available and preferred over using `lodash/get`. See https://github.com/getsentry/frontend-handbook#new-syntax for more information',
-          },
-          {
-            name: 'sentry/utils/theme',
-            importNames: ['lightColors', 'darkColors'],
-            message:
-              "'lightColors' and 'darkColors' exports intended for use in Storybook only. Instead, use theme prop from emotion or the useTheme hook.",
-          },
-          {
-            name: 'react-router',
-            importNames: ['withRouter'],
-            message:
-              "Use 'useLocation', 'useParams', 'useNavigate', 'useRoutes' from sentry/utils instead.",
-          },
-          {
-            name: 'sentry/utils/withSentryRouter',
-            importNames: ['withSentryRouter'],
-            message:
-              "Use 'useLocation', 'useParams', 'useNavigate', 'useRoutes' from sentry/utils instead.",
-          },
-          {
-            name: 'qs',
-            message: 'Please use query-string instead of qs',
-          },
-          {
-            name: 'moment',
-            message: 'Please import moment-timezone instead of moment',
-          },
-        ],
-      },
-    ],
-
-    // TODO(@anonrig): Remove this from eslint-sentry-config
-    'space-infix-ops': 'off',
-    'object-shorthand': 'off',
-    'object-curly-spacing': 'off',
-    'import/no-amd': 'off',
-    'no-danger-with-children': 'off',
-    'no-fallthrough': 'off',
-    'no-obj-calls': 'off',
-    'array-bracket-spacing': 'off',
-    'computed-property-spacing': 'off',
-    'react/no-danger-with-children': 'off',
-    'jest/no-disabled-tests': 'off',
```

Comment:
these are all set to `error` now.

Reply:
i think they were disabled because biome covers these rules. Now we will have two linters doing the same thing

Reply:
We did yeah. I enabled them to get rid of the duplication in this file and none of them sound like slow checks to add. If someone wanted to argue that eslint was too slow to run before this PR, then it should be taking about the same amount of time as well.

Reply:
i'm more saying you need to update all the biome ignore with additional eslint ignores. try looking at this with the updated eslint rule https://github.com/getsentry/sentry/blob/master/static/app/views/issueList/customViewsHeader.spec.tsx#L729

Reply:
oh.. hrm github ci should've caught these then.

Reply:
touched a tsx file to trigger eslint. expecting to see 2 lines inside customViewsHeader.spec.tsx fail this rule.

Reply:
i bet the rules don't match between sentry and getsentry

Reply:
they basically do! I made an earlier PR to this sentry file that didn't make it into getsentry yet, but they will match 100% with this https://github.com/getsentry/getsentry/pull/15980

and i'll keep it updated with whatever happens in here.